lib/base-controller.js

```diff
@@ -23,27 +23,42 @@ function getErrorLength() {
   }
 }
 
-BaseController.prototype.getNextStep = function getNextStep(req, res) {
-  var next = Controller.prototype.getNextStep.apply(this, arguments);
+BaseController.prototype.saveValues = function saveValues(req) {
   var forks = (this.options || {}).forks || [];
 
   _.each(forks, function eachFork(fork) {
```

Comment:
I think you're probably correct. We should make this bug fix available without requiring that users have es6 support